draft-ietf-webtrans-http2.md

```diff
@@ -923,6 +993,22 @@ WebTransport streams. The signal is intended for the application using
 WebTransport, which is expected to attempt to gracefully terminate the session
 as soon as possible.
 
+## Capsule Ordering and Reliability
+
+The use of an ordered and reliable transport means that a receiver does not need
+to tolerate capsules that arrive out of order. This differs from QUIC in that a
+receiver is required to treat the arrival of out of order capsules rather than
```

Comment:
Capsules are strongly ordered in QUIC, they're delivered on the request stream

```suggestion
receiver is required to treat the arrival of out of order frames rather than
```